src/components/organisms/Badges/Badges.tsx

```diff
@@ -108,23 +126,7 @@ export const Badges: React.FC<{
       </div>
       <div className="badges-container">
         <div className="badges-title">{badges.length} Badges</div>
-        <ul className="badge-list">
-          {badges.map(
-            (b) =>
-              b && (
-                <li className="badge-list-item" key={b.label}>
-                  <Link to={getLocationLink(b.venue, b.room)}>
-                    <img
-                      className="badge-list-item-image"
-                      src={b.image}
-                      alt={`${b.label} badge`}
-                    />
-                  </Link>
-                </li>
-              )
-            // label missing on hover - similar to playa
-          )}
-        </ul>
+        <ul className="badge-list">{badgeList}</ul>
```

Comment:
[nit] I personally probably would have kept the `ul` in the `badgeList` const/memo as well